src/bindings/capi/thorvg_capi.h

```diff
@@ -127,6 +127,23 @@ typedef enum {
 } Tvg_Composite_Method;
 
 
+/**
+ * \brief Enumeration indicating the ThorVG class type.
+ *
+ * \ingroup ThorVGCapi_Paint
+ *
+ * \since 0.9
+ */
+typedef enum {
+    TVG_TYPE_IDENTIFIER_UNDEF = 0,   ///< Undefined type.
+    TVG_TYPE_IDENTIFIER_SHAPE,       ///< A shape type paint.
+    TVG_TYPE_IDENTIFIER_SCENE,       ///< A scene type paint.
+    TVG_TYPE_IDENTIFIER_PICTURE,     ///< A picture type paint.
+    TVG_TYPE_IDENTIFIER_LINEAR_GRAD, ///< A linear gradient type.
+    TVG_TYPE_IDENTIFIER_RADIAL_GRAD  ///< A radial gradient type.
+} Tvg_Type_Identifier;
```

Comment:
What do you think about removing the word "TYPE" from the enumeration name (resulting in "Tvg_Identifier")?
That would improve consistency.

Reply:
I prefer to keep it, otherwise it's an indicator of the whole thorvg? doesn;t make sense. now it's clear that it's an indicator of a type, but as you wish I can change it.

Reply:
Could you please tell us what practical problems you are concerned about? If you feel like it doesn't make sense. We can discuss it further. Personally, I don't see any logical problems either way. But according to your argument, all should have "TYPE" as well. 

`
Tvg_Result
Tvg_Composite_Method
Tvg_Path_Command
Tvg_Stroke_Cap
Tvg_Stroke_Join
Tvg_Stroke_Fill
Tvg_Fill_Rule
`

In my experience, naming is often a matter of personal preference, and many people have different preferences. Therefore, consistency should be the top priority criterion.

Reply:
And yes, the identifier covers a whole set of classes in ThorVG's like Tvg_Result.

Reply:
changed :)